modules/local/mhcnuggets/main.nf

```diff
@@ -0,0 +1,32 @@
+process MHCNUGGETS {
+    label 'process_single'
+    tag "${meta.sample}"
+
+    conda "bioconda::mhcnuggets=2.4.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mhcnuggets:2.4.0--pyh7cba7a3_0' :
+        'quay.io/biocontainers/mhcnuggets:2.4.0--pyh7cba7a3_0' }"
+
+    input:
+    tuple val(meta), path(tsv)
+
+    output:
+    tuple val(meta), path("*{_predicted_mhcnuggets.csv,_predicted_mhcnuggetsii.csv}"), emit: predicted
```

Comment:
How many files are comming out of this function?`
*predicted*mhcnug*.csv is also not working (worst case scenario?)

Reply:
Ah, nevermind then 👍